src/wifi.rs

```diff
@@ -327,6 +332,30 @@ pub struct WifiDriver<'d> {
     _p: PhantomData<&'d mut ()>,
 }
 
+#[cfg(feature = "async")]
```

Comment:
None of these are necessary. `async fn` is supported on the stable compiler for structs. Remove *ALL* of these type aliases.